src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
You should follow BEM naming style here - you can use underscore only for once (because after underscore comes an element, and by BEM there can be only ONE element. Here you have TWO). You could change name of this class to 'navigation__items-list-item'(without one underscore) or just 'item' to start a new block. Pay attention that this class repeats below in the code - fix it in all places of its appearance.
```suggestion
          <li class="navigation__items-list-item">
```